release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -573,6 +580,17 @@ For more information, see xref:../machine_management/creating_machinesets/creati
 
 The Machine API now honors cluster-wide proxy settings. When a cluster-wide proxy is configured, all Machine API components will route traffic through the configured proxy.
 
+[id="ocp-4-7-mco-auto-reboot-triggers-removed"]
+==== Some machine configuration updates no longer cause automatic reboot
+
+The Machine Config Operator (MCO) no longer automatically reboots all corresponding nodes for the following machine configuration changes:
+
+* changes to the SSH key in the `spec.config.ignition.passwd.users.sshAuthorizedKeys` parameter of a machine config
+* changes to the global pull secret or pull secret in the `openshift-config` namespace
+* changes to the `/etc/containers/registries.conf` file, such as adding or editing an `ImageContentSourcePolicy` object
+
+For more information, see xref:../architecture/control-plane.adoc/understanding-machine-config-operator_control-plane[Understanding the Machine Config Operator].
```

Comment:
.adoc#understanding-machine-config-operator_control-plane[Understanding the Machine Config Operator].